### PR TITLE
[WIP] Bandage pets

### LINKED ISSUE #62141: ability to treat monster wounds
### Is your feature request related to a problem? Please describe.

Currently hurt pets and mounts slowly regenerate health, but if they are bleeding or just not healing fast enough there is nothing you can do to save your friend or get them back to full health

### Solution you would like.

Ability to bandage, disinfect etc on monsters in the same way the player can do this to themself or NPCs

### Describe alternatives you have considered.

Just use Magiclysm or Arcana to do this with powerful magics

### Additional context

Inspired especially by recent Rycon and Critsy Bear videos which both dealt with this issue, either by repeated vicious pet murder or magical powers

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
Attempts to add bandaging pets as an option.  May fix #62141

#### Describe the solution
Adds a function to bandage pets and remove the bleed effect in monexamine.cpp

#### Describe alternatives you've considered
Letting someone who knows what they are doing respond.

### CODE REVIEW

**Comment by fungamer2-2** [comment]
The check failures are saying that you need to declare `effect_bleed` at the top of the file

File: src/monexamine.cpp [hunk 1]
```diff
@@ -101,6 +101,21 @@ void remove_saddle_from( monster &z )
...
+{
+    if( z.has_effect( effect_bleed ) ) {
+        return;
+    }
+    item_location loc = bandage_loc();
+    if( !loc ) {
+        add_msg( _( "Never mind." ) );
+        return;
+    }
+    z.remove_effect( effect_bleed, 1_turns, true );
```
Comment (fungamer2-2) [inline src/monexamine.cpp:114]:
```suggestion
    z.remove_effect( effect_bleed );
```

File: src/monexamine.cpp [hunk 2]
```diff
@@ -101,6 +101,21 @@ void remove_saddle_from( monster &z )
     z.tack_item.reset();
 }
 
+void bandage_animal( monster &z )
+{
+    if( z.has_effect( effect_bleed ) ) {
```
Comment (fungamer2-2) [inline src/monexamine.cpp:106]: `effect_bleed` also needs to be declared at the top of the file.

Comment (Maleclypse) [reply src/monexamine.cpp:106]: Thanks! I haven't had a chance to get back to this one after opening it.

Comment (fungamer2-2) [inline src/monexamine.cpp:106]:
```suggestion
    if( !z.has_effect( effect_bleed ) ) {
```

**Comment by Termineitor244** [comment]
This PR will fix #56878 when merged, thanks!

**Comment by Zireael07** [comment]
Probably worth putting in the "to revive" list?

**Comment by Termineitor244** [comment]
> Probably worth putting in the "to revive" list?

It's definitely worth it.

**Comment by Maleclypse** [comment]
Yeah, I just realized it's still outside of my skills to complete and I have a bunch of open projects.  I'll add to the abandoned but desired list

**Pull request closed without merging**